src/ui/mainWindow.ts

```diff
@@ -37,7 +37,7 @@ model._selectedRide.subscribe(r =>
 
 export const mainWindow = window({
 	title,
-	width: { value: 500, min: 465, max: 560 },
+	width: 500, minWidth: 465, maxWidth: 560,
```

Comment:
It seems you are reverting some changes from a more recent version here? This file should probably also not be in this PR as it doesn't add anything related to the renaming functionality.

Reply:
Corrected it, I didn't update flexui before continuing in my own fork.